content/docs/components/jupyter.md

```diff
@@ -4,6 +4,18 @@ description = "Using Jupyter notebooks in Kubeflow"
 weight = 10
 +++
 
+## Creating a custom Jupyter image
```

Comment:
I think this section should appear lower down on the page. The section on bringing up the Jupyter notebook within Kubeflow is more important to most people.

Reply:
Done.